.werft/jobs/build/job-config.ts

```diff
@@ -78,6 +78,7 @@ export function jobConfig(werft: Werft, context: any): JobConfig {
     const withContrib = "with-contrib" in buildConfig || mainBuild;
     const withPreview = "with-preview" in buildConfig && !mainBuild;
     const storage = buildConfig["storage"] || "";
+    const selfSigned = "self-signed" in buildConfig && !mainBuild;
```

Comment:
From what I've been discussing with other teams (workspace team specifically) about `self-signed` support, A lot of them seem to need a direct way to test and removing this option essentially makes it harder.

Without this flag, I'm not even sure if we would also use this on our support experiments. Hence, I'd prefer to have it.

Reply:
This flag has no effect right now. If you want to add it to the preview setup, you have to use these flags like [done here](https://github.com/gitpod-io/gitpod/blob/main/.werft/jobs/build/self-hosted-upgrade-tests.ts#L77).  The way it is right now, it is just dead code. For which part of the current implementation do you think this code is necessary?

Reply:
I'm thinking this is required, to trigger the workflow with `selfSigned` option i.e for a command like

```
werft run github -f -s .werft/installer-tests.ts -j .werft/eks-installer-tests.yaml -a debug=true -a self-signed=true -a skipTests=true -a preview=true
```

to work? Isn't that the case? 🤔

Reply:
`job-config.ts` deals with werft flags used when running `build.yaml`. In the case of comments in PRs for example. If the usecase here is to use `self-signed` annotation with the YAML files `*-installer-tests.yaml`, this part is unnecessary. If the goal is to add it to the self-hosted preview setup (i.e. comment via pull requests), you have to make the changes [here too](https://github.com/gitpod-io/gitpod/blob/main/.werft/jobs/build/self-hosted-upgrade-tests.ts#L77). 

I recommend not doing the latter for now mostly because self-signed certs functionality do not work in this setup now anyways due to https://github.com/gitpod-io/gitpod/issues/11811. Wouldn't it be wise to add it once that issue is fixed?